src/intersection-observer-polyfill.js

```diff
@@ -429,6 +438,21 @@ export class IntersectionObserverPolyfill {
     return;
   }
 
+  /**
+   * Handle Mutation Observer Pass
+   * This performas the tick, and is wrapped in a paas
+   * To handle throttling of the observer
+   * @private
+   */
+  handleMutationObserverPass_() {
+    if (this.viewport_ && this.resources_) {
+      this.resources_.onNextPass(() => {
+        this.tick(this.viewport_.getRect());
```

Comment:
`tick` still needs to be throttled.

Reply:
Tick is being throttled by the pass isn't it? 😮 Or do you want the pass inside the tick itself?

Reply:
Oh, I totally forgot we were using a Pass. This is fine, then.